.gitlab-ci.yml

```diff
@@ -19,7 +19,8 @@ workflow:
     - if: $CI_COMMIT_BRANCH
 
 .docker-env:                       &docker-env
-  image:                           paritytech/ink-ci-linux:production
+  # Temporary image, remove when clang in "ink-ci-linux:production" is updated to 13
```

Comment:
Is that still true? Don't you actually just created this image for the future?

Reply:
It depends on how quickly clang in ci image for `ink` would be updated. 
If it takes too much time I'll recreate this image with `production` tag and setup it's updates. So currently it is just a quick fix

Reply:
Do you keep track of this?

Reply:
I'll create the issue